src/main/java/com/vite/service/ViteClient.java

```diff
@@ -0,0 +1,59 @@
+package com.vite.service;
+
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TBinaryProtocol;
+import org.apache.thrift.protocol.TProtocol;
+import org.apache.thrift.transport.TSocket;
+import org.apache.thrift.transport.TTransport;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PreDestroy;
+
+/**
+ * Simple client for proof of concept.
+ * TODO define interface
+ * TODO break out into own repo
+ * TODO use secure client
+ */
+public class ViteClient {
```

Comment:
We don't publish this currently, and this is going to be in its own repo, so I am not going to figure that out to just undo it